client/ayon_resolve/plugins/load/load_editorial_package.py

```diff
@@ -1,11 +1,15 @@
+import json
 from pathlib import Path
+import random
 
 from ayon_core.pipeline import (
+    AVALON_CONTAINER_ID,
```

Comment:
You should use `AYON_CONTAINER_ID`, but looking in the code, you coudl not add it to the metadata at all, and just make sure `"id"` is filled when you return in `list_containers`. The same for `AYON_INSTANCE_ID` btw.

Reply:
And also use `ayon:container-3.0` instead of `openpype:container-2.0`.